tests/calculations/test_pp.py

```diff
@@ -79,6 +81,14 @@ def test_pp_keep_plot_file(aiida_profile, fixture_sandbox, generate_calc_job, ge
         assert element in calc_info.retrieve_list
 
 
+def test_pp_cmdline_setting(aiida_profile, fixture_sandbox, generate_calc_job, generate_inputs):
+    """Test a `PpCalculation` with user-defined cmdline setting."""
+    entry_point_name = 'quantumespresso.pp'
+    inputs = generate_inputs(settings={'cmdline': ['-npools', '2']})
+    calc_info = generate_calc_job(fixture_sandbox, entry_point_name, inputs)
+    assert ['-npools', '2'] == calc_info.codes_info[0].cmdline_params
```

Comment:
```suggestion
    assert calc_info.codes_info[0].cmdline_params == ['-npools', '2']
```
This is a nitpick, but since you have to make a small update in the `_generate_inputs` anyway, I though you could include this as well. I think when testing it reads more natural to put the unknown value that you are testing as the first operand.